src/gocept/gtimelog/core.py

```diff
@@ -570,6 +575,9 @@ def load(self, filename):
         self.engagement = config.get('gtimelog', 'engagement')
         if self.engagement:
             self.engagement = [int(e) for e in self.engagement.split(',')]
+        self.holidays = config.get('gtimelog', 'holidays')
```

Comment:
So you expect the holidays now in the config?
Do we have documentation to tell this?

Reply:
I have added something to the readme.